### PR TITLE
Add the APT repository

### PR DESCRIPTION
**Pull request by toothbrush**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,22 +24,22 @@ should be usable next time you reload your shell config.
...
-sudo apt install <filename>
-```
-
-or
-
-```
-sudo dpkg -i <filename>
+```sh
+sudo apt install apt-transport-https curl gnupg2
+# This is the Delivery Engineering GPG key, to verify releases:
```
Comment (rbvigilante) [inline README.md]: This should probably be the 'redbubble deb repo key' or something; we're leaking corporate structure here :P

Comment (toothbrush) [reply README.md]: Meh yeah i guess.  I'm a bit too lazy to regenerate the key, stick it in LastPass, etc etc..

File: README.md [hunk 2]
```diff
@@ -24,22 +24,22 @@ should be usable next time you reload your shell config.
 
 #### Ubuntu/Debian (`deb` package)
 
-Download the DEB package from the [latest release](https://github.com/redbubble/yak/releases/latest) page.
+There's a shiny repository available for your appreciation and
```
Comment (rbvigilante) [inline README.md]: Maybe just, like, "`yak` can be installed from our ppm:"?

**Comment by toothbrush** [comment]
[{yak} make it easy to keep up-to-date on Linux](https://trello.com/c/dsUGHoEu/3025-yak-make-it-easy-to-keep-up-to-date-on-linux)

**Pull request merged**